PF1/assignment5/CheckArraySorting.java

```diff
@@ -0,0 +1,63 @@
+package assignment5;
+
+/* 0: unsorted
+ * 1: ascending sorted
+ * 2: descending sorted
+ * */
+public class CheckArraySorting {	
+	public int checkSortArray(int[] a){
+		if(a==null)
+			throw new NullPointerException();
+		if(a.length <= 0)
+			throw new ArrayIndexOutOfBoundsException();
+				
+		/*check for ascending */
+		int i=0;
+		for(i=0; i< a.length-1 ; i++){
+			if(a[i] > a[i+1]){
+				break;
+			}
+		}
+				
+		/* => is in ascending order*/
+		if(i == a.length -1){
+			return 1;
+		}
+				
+		/*descending check */
+		for(i=0; i< a.length -1; i++){
+			if(a[i] < a[i+1])
+				break;
+		}
+			
+		/* => is descending sorted*/
+		if(i == a.length- 1){
+			return 2;	 
+		}
+			
+		return 0;
+		}
+}			
+		
+/**		Another algorithm
```

Comment:
This code included different approach to same problem solving.
All extra comments have been removed.